src/System.Runtime/tests/System/StringsMatchingNonRandomizedHashCode.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Tests
+{
+    internal static class StringsMatchingNonRandomizedHashCode
+    {
+        /// <summary>
+        /// A list of strings that all have the same hashcode (-1038491930) using the legacy non randomized string hash code algorithm
+        /// </summary>
+        internal static readonly IEnumerable<string> Data = Environment.Is64BitProcess ? _64bit : _32bit;
```

Comment:
All static fields contained within a single type are initialized together. I am not sure whether it is guaranteed that  _64bit / _32bit fields are initialized here.  And even if the order is guaranteed somewhere - it is pretty fragile to write the code this way.

You can fix this by changing Data into property.